### ISSUE TITLE
Slide to answer a call

### ISSUE THREAD

**Issue by DodoLeDev** [Chooloo/koler#376]
### Checklist

- [x] I am using the latest version - 1.4.3 
- [x] I checked, but didn't find any duplicates (open OR closed) of this issue in the repo. 
- [x] This issue contains only one feature request. I will open one issue for every feature I want to request.

#### Describe the feature you want
When you receive an incoming call while your device is off, Koler should let us answer it by **sliding** the answer button, not just by clicking on it.

#### Is your feature request related to a problem? Please describe it
When you receive a call while your phone is in your pocket, you are more likely to awkwardly answer the call if you just have to click a button instead of sliding!

#### Additional context
And because a picture is worth a thousand words, I have made a little homemade animation of what I would like to have in a future version of Koler!
![SlideToAnswer](https://user-images.githubusercontent.com/59477313/159186048-08773729-3d4f-48c0-a96d-870c01271f92.gif)

#### How will you/everyone benefit from this feature?
Adding this feature will greatly decrease the chances to awkwardly answer an incoming call!

**Comment by Pranab009** [comment]
This is too good add this feature. But vertical swipe.

**Comment by Ilithy** [comment]
Duplicate, please see : https://github.com/Chooloo/koler/issues/215

_Otherwise indeed, I liked this little feature._

**Comment by bagajohny** [comment]
Yes, this is essential. Please add swipe up to answer/reject calls.
